tools/Explicit.NuGet.Versions/Explicit.NuGet.Versions.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net45</TargetFramework>
+		<OutputPath>build\</OutputPath>
+		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+		<AssemblyName>nev</AssemblyName>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="DotNetZip" Version="1.10.1" />
```

Comment:
I assume `System.IO.Compression` isn't up to the job for some reason?

Reply:
The ZipFile object expects you to extract the entire contents and repack it. This allows me to do it by doing an in-place swap of the nuspec file.